Druid/DRJitCompileTimeExpression.class.st

```diff
@@ -15,6 +15,34 @@ DRJitCompileTimeExpression class >> expression: anExpression [
 		yourself
 ]
 
+{ #category : #sccp }
+DRJitCompileTimeExpression class >> tryParseOperandForSCCP: aDRValue [
```

Comment:
Why do we need this?

I mean.

If we run the canonicaliser first, we could make it make all constants "fall" on one side of the tree, to help with folding.
Then these cases here would be simplified by A LOT

Reply:
Yes, that could simplify. Anyway, we'll need these methods to transform constants (or other instructions) to be operated.